src/preferences/dialog/dlgprefwaveform.cpp

```diff
@@ -559,6 +561,12 @@ void DlgPrefWaveform::slotSetWaveformOverviewType() {
 
 void DlgPrefWaveform::slotSetDefaultZoom(int index) {
     WaveformWidgetFactory::instance()->setDefaultZoom(index + 1);
+    for (int i = 1; i <= PlayerManager::numDecks(); i++) {
```

Comment:
```suggestion
    for (unsigned int i = 1; i <= PlayerManager::numDecks(); i++) {
```

Reply:
Yep, saw that. I've also included samplers update.